guava/pom.xml

```diff
@@ -37,6 +37,10 @@
       <artifactId>animal-sniffer-annotations</artifactId>
       <version>${animal.sniffer.version}</version>
     </dependency>
+    <dependency>
+      <groupId>tech.harmonysoft</groupId>
+      <artifactId>traute-javac</artifactId>
```

Comment:
What's this for?

Reply:
@liach this adds *Traute* *javac* plugin to compiler's classpath